src/ScottPlot/ScottPlot.csproj

```diff
@@ -22,9 +22,9 @@ Release cookbook: https://swharden.com/scottplot/cookbook
 Release demo: https://swharden.com/scottplot/demo</PackageReleaseNotes>
     <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
-    <Version>4.1.16</Version>
-    <AssemblyVersion>4.1.16.0</AssemblyVersion>
-    <FileVersion>4.1.16.0</FileVersion>
+    <Version>4.1.17</Version>
```

Comment:
I think Scott will bump the version himself once this is merged and any other PRs have been merged in.

Reply:
Indeed! This can be left as `4.1.16` for now 👍

Reply:
I didn't intentionally change the version number, probably because I'm not familiar with ScottPlot's automated build system. I also think this small change should not change the version number.